### PR TITLE
Add helper pods + support for data movement test

### PR DESCRIPTION
**Pull request by bdevcich**
- Added /helper_image to support test setup/verification scripts
- Added workflow to build helper image
- Added HardwareRequired decorator to only allow tests to run in
  non-KIND contexts
- Added raw simple test

Signed-off-by: Blake Devcich <blake.devcich@hpe.com>

### CODE REVIEW

**Review by roehrich-hpe — APPROVED** [review]

File: helper_image/scripts/copy-out.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+function usage() {
+    echo
+    echo "Verify the md5sums of two files match"
+    echo "This is used after copy_out directives to ensure the copy_in file matches the copy_out file"
+    echo
+    echo "Syntax: copy-out.sh FILE_IN FILE_OUT"
+    echo
+    echo "examples:"
+    echo "copy-out.sh testuser/test.in testuser/test.out"
```
Comment (matthew-richerson) [inline helper_image/scripts/copy-out.sh:11]: Are these relative paths? How does it know where the lustre mount is?

**Review by matthew-richerson — APPROVED** [review]

File: helper_image/scripts/copy-in.sh [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    echo "This is used prior to copy_in directives to ensure the FILEPATH/directory are in"
+    echo "place."
+    echo
+    echo "Syntax: copy-in.sh ROOT_DIR FILEPATH UID GID"
+    echo
+    echo "arguments:"
+    echo "ROOT_DIR      root path directory; assume this is the global lustre"
+    echo "              mount root (e.g. /lus/global)"
+    echo "FILEPATH      filepath to the FILEPATH to create. This should be a"
+    echo "              path (e.g. /testuser/test.in, test.in)"
```
[the remark this answers is not in the document]
Comment (bdevcich) [reply helper_image/scripts/copy-in.sh:17]: Yes

File: helper_image/scripts/copy-in.sh [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    echo "UID           user ID of the filepath"
+    echo "GID           groupd ID of the filepath"
```
[the remark this answers is not in the document]
Comment (bdevcich) [reply helper_image/scripts/copy-in.sh:19]: Yes

File: helper_image/scripts/copy-out.sh (hunk 1 above)
Comment (bdevcich) [reply helper_image/scripts/copy-out.sh:11]: They should be absolute and say: `/lus/global/testuser/test.in`

**Pull request merged**